hosted_control_planes/hcp-backup-restore-dr.adoc

```diff
@@ -27,6 +27,9 @@ The following procedures are specific to hosted control planes on {aws-short}.
 :FeatureName: Hosted control planes on the {aws-short} platform
 include::snippets/technology-preview.adoc[]
 
+:FeatureName: Hosted control planes on AWS
```

Comment:
The TP snippet seems to be showing up twice, unless these are different things?

![Screenshot 2023-11-21 at 12 45 16 PM](https://github.com/openshift/openshift-docs/assets/96192119/3bc05129-7460-4e72-9641-0c3512cc8695)

Reply:
oh weird! I'll take a look.